src/datastore.cpp

```diff
@@ -291,8 +296,12 @@ b_fs::path datastore::get_parent_path(t_datastore file_type,
 	return parent_path;
 }
 
+std::string datastore::home_dir;
+
+void datastore::set_home(const std::string &home) {home_dir = home;}
```

Comment:
I would add

_note("Setting datastore home=" << home);

it's important place and possible sources of errors to troubleshoot